web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -55,8 +55,8 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
     });
   };
 
-  const toggleAnnotations = () => {
-    setShowAnnotations({ checked: !showAnnotations.checked });
+  const toggleAnnotations = (checked: boolean) => {
```

Comment:
I think at this point you can delete this function and better inline it :smile:

Reply:
good point, will do